plugins/hooks/frontend/public/templates/vue-drawer.html

```diff
@@ -16,9 +17,16 @@
                     </el-input>
                 </cly-form-field>
 
-                <cly-form-field name="description" :label="i18n('hooks.description')">
-                    <div class="cly-vue-hook-drawer__help-text">
-                        {{i18n('hooks.optional')}}
+                <cly-form-field name="description">
```

Comment:
cly-form-field accepts label and optional as prop. So you did not have to write your custom template for it unless its different.

Reply:
great, updated